### PR TITLE
Add repartition warnings for Nexus 4 and 7

### PR DESCRIPTION
**Pull request by yagyaxt1034**

### CODE REVIEW

File: v2/devices/mako.yml [hunk 1]
```diff
@@ -12,6 +12,10 @@ user_actions:
     title: "Reboot to Bootloader"
     description: "Press and hold the volume buttons and the power button until the phone reboots."
     button: true
+  confirm_model:
```
Comment (Flohack74) [inline v2/devices/mako.yml:15]: You also need to include that message somewhere, e.g. in the unlock steps array.

File: v2/devices/mako.yml [hunk 2]
```diff
@@ -12,6 +12,10 @@ user_actions:
...
+    title: "Confirm your model"
+    description: 'Note, that issues can be encountered while flashing Nexus 4 (mako) due to altered partition layouts. If you''ve previously had a recent version of LineageOS installed (>16.0, July 2019 or later) the partition layout has probably been altered. See https://github.com/ubports/ubports-installer/issues/1895 for instructions on how to revert to the original partition layout. If you''re unsure whether this is the case you may go ahead now and revisit the instructions linked above if you encounter "Error: systemimage: Push failed: Failed push: Failed to push file 0: Push failed: out of space" while flashing the device.'
```
Comment (NeoTheThird) [inline v2/devices/mako.yml:17]:
```suggestion
    description: 'If a recent version of LineageOS (>16.0, July 2019 or later) was installed before, the partition layout might have been altered, and you will have to revert to the original partition layout. If you are unsure whether this is the case, you may go ahead now and revisit the instructions linked above if you encounter "Error: systemimage: Push failed: Failed push: Failed to push file 0: Push failed: out of space" while flashing the device.'
    link: 'https://github.com/ubports/ubports-installer/issues/1895'
```

File: v2/devices/mako.yml [hunk 3]
```diff
@@ -12,6 +12,10 @@ user_actions:
...
+
```
Comment (NeoTheThird) [inline v2/devices/mako.yml:18]:
```suggestion
```

File: v2/devices/mako.yml (hunk 1 above)
Comment (NeoTheThird) [reply v2/devices/mako.yml:15]:
```suggestion
  partition_layout:
```

File: v2/devices/mako.yml [hunk 4]
```diff
@@ -12,6 +12,10 @@ user_actions:
     title: "Reboot to Bootloader"
     description: "Press and hold the volume buttons and the power button until the phone reboots."
     button: true
+  confirm_model:
+    title: "Confirm your model"
```
Comment (NeoTheThird) [inline v2/devices/mako.yml:16]:
```suggestion
    title: "Partition Layout"
```

File: v2/devices/flo.yml [hunk 5]
```diff
@@ -16,7 +16,7 @@ user_actions:
     button: true
   confirm_model:
     title: "Confirm your model"
-    description: "Please double-check that your device is an LG Nexus 7 2013 WiFi (flo). If you have the LTE version (deb), you have to select that instead. The 2012 version is not compatible."
+    description: 'Please double-check that your device is an LG Nexus 7 2013 WiFi (flo). If you have the LTE version (deb), you have to select that instead. The 2012 version is not compatible. Note, that issues can be encountered while flashing Nexus 7 (deb/flo) due to altered partition layouts. If you''ve previously had a recent version of LineageOS installed (>16.0, July 2019 or later) the partition layout has probably been altered. See https://github.com/ubports/ubports-installer/issues/1895 for instructions on how to revert to the original partition layout. If you''re unsure whether this is the case you may go ahead now and revisit the instructions linked above if you encounter "Error: systemimage: Push failed: Failed push: Failed to push file 0: Push failed: out of space" while flashing the device.'
```
Comment (NeoTheThird) [inline v2/devices/flo.yml:19]:
```suggestion
    description: 'Please double-check that your device is an LG Nexus 7 2013 WiFi (flo). If you have the LTE version (deb), you have to select that instead. The 2012 version is not compatible.'
  partition_layout:
    title: "Partition layout"
    description: 'If a recent version of LineageOS (>16.0, July 2019 or later) was installed before, the partition layout might have been altered, and you will have to revert to the original partition layout. If you are unsure whether this is the case, you may go ahead now and revisit the instructions linked above if you encounter "Error: systemimage: Push failed: Failed push: Failed to push file 0: Push failed: out of space" while flashing the device.'
    link: 'https://github.com/ubports/ubports-installer/issues/1895'
```

File: v2/devices/deb.yml [hunk 6]
```diff
@@ -16,7 +16,7 @@ user_actions:
     button: true
   confirm_model:
     title: "Confirm your model"
-    description: "Please double-check that your device is an LG Nexus 7 2013 LTE (deb). If you have the wifi-only version (flo), you have to select that instead. The 2012 version is not compatible."
+    description: 'Please double-check that your device is an LG Nexus 7 2013 LTE (deb). If you have the wifi-only version (flo), you have to select that instead. The 2012 version is not compatible. Note, that issues can be encountered while flashing Nexus 7 (deb/flo) due to altered partition layouts. If you''ve previously had a recent version of LineageOS installed (>16.0, July 2019 or later) the partition layout has probably been altered. See https://github.com/ubports/ubports-installer/issues/1895 for instructions on how to revert to the original partition layout. If you''re unsure whether this is the case you may go ahead now and revisit the instructions linked above if you encounter "Error: systemimage: Push failed: Failed push: Failed to push file 0: Push failed: out of space" while flashing the device.'
```
Comment (NeoTheThird) [inline v2/devices/deb.yml:19]:
```suggestion
    description: 'Please double-check that your device is an LG Nexus 7 2013 LTE (deb). If you have the wifi-only version (flo), you have to select that instead. The 2012 version is not compatible.'
  partition_layout:
    title: "Partition layout"
    description: 'If a recent version of LineageOS (>16.0, July 2019 or later) was installed before, the partition layout might have been altered, and you will have to revert to the original partition layout. If you are unsure whether this is the case, you may go ahead now and revisit the instructions linked above if you encounter "Error: systemimage: Push failed: Failed push: Failed to push file 0: Push failed: out of space" while flashing the device.'
    link: 'https://github.com/ubports/ubports-installer/issues/1895'
```

**Review by NeoTheThird — CHANGES_REQUESTED** [review]
grammar and code style fixes. this should have its own user_action and belongs in the os-prerequisites, not in the general unlock steps.

**Pull request merged**